wicket-native-websocket/wicket-native-websocket-core/src/main/java/org/apache/wicket/protocol/ws/api/BaseWebSocketBehavior.java

```diff
@@ -120,9 +120,21 @@ protected String getWebSocketSetupScript(Map<String, Object> parameters) {
 		return webSocketSetupTemplate.asString(parameters);
 	}
 
+	/**
+	 * Override to return a context. By default, this is the page class name.
+	 *
+	 * @param component the {@link org.apache.wicket.Component}
+	 * @return the context for this websocket behavior.
+	 */
+	protected String getContext(Component component) {
+		return component.getPage().getClass().getName();
+	}
+
 	private Map<String, Object> getParameters(Component component) {
 		Map<String, Object> variables = Generics.newHashMap();
 
+		variables.put("context", component.getPage().getClass().getName());
```

Comment:
Use `getContext(component)`

Reply:
Thanks for spotting this. Added factory method and forgot referencing here